src/js/components/deployments/deployment-report/phaseprogress.js

```diff
@@ -0,0 +1,249 @@
+import React, { useState } from 'react';
+import moment from 'moment';
+import momentDurationFormatSetup from 'moment-duration-format';
+import pluralize from 'pluralize';
+
+import { Button } from '@material-ui/core';
+import { CheckCircle, ErrorRounded, Pause, PlayArrow, Warning as WarningIcon } from '@material-ui/icons';
+
+import { deploymentDisplayStates, deploymentStatesToSubstates, pauseMap } from '../../../constants/deploymentConstants';
+import { getDeploymentState, groupDeploymentStats, statCollector } from '../../../helpers';
+import Confirm from '../../common/confirm';
+import theme, { colors } from '../../../themes/mender-theme';
+import inprogressImage from '../../../../assets/img/pending_status.png';
+
+momentDurationFormatSetup(moment);
+
+const shortCircuitIndicators = ['already-installed', 'noartifact'];
+const substateMap = {
+  download: {
+    title: 'download',
+    done: 'downloaded',
+    successIndicators: ['installing', 'rebooting', ...deploymentStatesToSubstates.paused, 'success'],
+    failureIndicators: deploymentStatesToSubstates.failures
+  },
+  install: {
+    title: 'install',
+    done: 'installed',
+    successIndicators: ['rebooting', 'pause_before_rebooting', 'pause_before_committing', 'success'],
+    failureIndicators: []
+  },
+  reboot: { title: 'reboot', done: 'rebooted', successIndicators: ['pause_before_committing', 'success'], failureIndicators: [] },
+  commit: { title: 'commit', done: 'comitted', successIndicators: deploymentStatesToSubstates.successes, failureIndicators: [] }
```

Comment:
@kacf here it is! - I'll fix it once I get to the next round of changes

Reply:
Aha, it is lowercase! That explains it.